### DISCUSSION TITLE
Lazy load and display placeholder until images are loaded

### DISCUSSION THREAD

**Discussion by ryangjchandler** [livewire/livewire#7539] [3. Feature ideas]
I'm currently using lazy loading & placeholder HTML to display placeholder image elements as the image URLs themselves are fetched from a third-party API that could be slow.

It definitely improves the UX as the page loads instantly, but the "janky" part is that once the placeholder is gone and the real component HTML is loaded, the images are still loading in and take a second to finish loading.

Is it possible to prevent the real markup from being diffed and patched into the DOM until after all of the images have been loaded in?

**Comment by ArtMin96** [comment]
I also have a problem when I use lazy and placeholder together. I fetch data from a third-party API and create a slide with a swiper. But when I use lazy and placeholder together, problems arise.
